data/outfits/god.txt

```diff
@@ -19,18 +17,80 @@ outfit "Ruin-The-Fun God Mode"
 	"corrosion protection" 1000000
 	"leak protection" 1000000
 	"burn protection" 1000000
-	bunks 1000000
 	description "Absolutely illegal."
 
+outfit "Ruin-The-Fun God Jammer"
+	"display name" "RTF God Jammer"
+	category "Special"
+	cost 0
+	thumbnail "outfit/large radar jammer"
+	"radar jamming" 1000000
+	"optical jamming" 1000000
+	"scan interference" 1000000
+	description "Allows you to jam a lot."
```

Comment:
To protect friendly ship, I imagine a powerful anti-missile turret that also has jamming, it would take no turret slots so even if it's not mounted due to not sufficient amount of turret mounts, you would still have the jamming.

You may or may not add such an outfit in this PR, if not ill make an issue because I think this would be a good one to have.